### ISSUE TITLE
Would it be possible to use the joystick as an xbox joystick for the pc?

### ISSUE THREAD

**Issue by raj-palan** [xqinx/xobdox#1]
I'm trying to build a gaming keypad with a joystick, and that requires that the analog thumbstick is seen as game controller joystick.
Thanks

**Comment by xqinx** [comment]
> I'm trying to build a gaming keypad with a joystick, and that requires that the analog thumbstick is seen as game controller joystick.
> 
> Thanks

Yes it's possible to add generic joystick support in general. What you will need is to report as joystick over HID instead of mouse. For Xbox specifically you might want to do a little bit research and see if they have their own tweaks on what to report.

**Comment by rp1231** [comment]
Thanks for your reply.
Found this https://github.com/badjeff/zmk-hid-io for hid output by asking on the zmk discord.
